hypnotoad/core/mesh.py

```diff
@@ -739,14 +748,17 @@ def surface_vec(lower):
                     p_out = c_out[c_out.endInd]
                 return [p_out.R - p_in.R, p_out.Z - p_in.Z]
 
-            if self.nonorthogonal_options.nonorthogonal_spacing_method == "orthogonal":
+            if (
```

Comment:
This big `if-elif` chain might be nicer as a `dict`, but definitely happy for that to be future work

Reply:
Agreed, but indeed should be part of a future PR, fixing #34